script/Arcueid/Day 7/07_00_ARC07_3_1_4.txt

```diff
@@ -262,6 +262,7 @@ You don't really have any talents apart from killing, so I'll do my utmost to su
 [sha:53ccbc825115723e3381b965c168b7570bb34a4e]{
 -- Page 641, Offset 18191.
 -- ……妙な自信をつけさせてしまった。
+// Group PR Mark: Switched on.. Triggered instead?
```

Comment:
Triggered doesn't really sit right with me. Activated? Fuelled?

Reply:
activated works for me